src/domain/eventSearch/utils.tsx

```diff
@@ -221,11 +221,13 @@ export const getEventSearchVariables = ({
     .map((category) => MAPPED_CATEGORIES[category])
     .filter((e) => e);
 
+  const hasDivisions = !isEmpty(divisions);
+  const hasText = !isEmpty(text);
   // Combine and add keywords
-
   return {
-    allOngoingAnd: text,
-    ...(!isEmpty(divisions) && { division: divisions.sort() }),
+    ...(hasText &&
```

Comment:
complex code but it ensures that the parameters are added only when they have nonempty value. This makes also mock writing easier when we dont have to expect null or undefined value or empty list.

Reply:
if his gets hard to read we could just create object in a variable and add these fields in if statements and then return the object

Reply:
Should internet events also be hidden when there is at least one place selected?

Reply:
And maybe also `division: ["kunta:helsinki"]` should be added to query when no other division is selected and `text` array is empty.

Reply:
> Should internet events also be hidden when there is at least one place selected?

Yes good idea I added the check

Reply:
> And maybe also `division: ["kunta:helsinki"]` should be added to query when no other division is selected and `text` array is empty.

I think it's redundant data because when `allOngoingAnd` is added, it searches from helsinki area by default and seemed to completely ignore division values.